packages/users-module/src/index.js

```diff
@@ -1 +1,2 @@
-module.exports = {};
\ No newline at end of file
+
+module.exports = {};
```

Comment:
This should be the opposite, `module.exports` at line 1, and line 2 empty.

Reply:
It looks great, by my side, only this change, and as soon as it's done I'll approve.![](https://media0.giphy.com/media/l0K4mbH4lKBhAPFU4/giphy.gif)